### PR TITLE
Review KPI

### PR DESCRIPTION
**Pull request by jsieland**
As per my TO DO from our last meeting I did a more in-depth review of the first KPI draft.

#### List of changes

- Overall:
    - put examples to dedicated subsection at the end
    - editorial review (fix line breaks, formatting, etc.)
    - change examples if not fitting KPI
    - rename KPI if unclear or confusing
- title
    - put GDC context to rationale
- description
    - changed last rule to be a "positive" 
- links -> links-health
    - rename KPI 
- providers -> distribution-providers
    - rename KPI 
    - rewrite text because unclear if provider or links related
    - changed example because old example did not had "distributor" provider
- temporal -> time-intervals
    - temporal extent is confusing because we have time and additionalExtents.temporal
    - basically this KPI relates to intervals, so rename to Time Intervals
    - updated rules in relation to time intervals (which are not already covered by WCMP2)
- validation -> wcmp2-compliance
    - rename to WCMP2-compliance
    - added guidance to not just transform old metadata ;)

#### Additional question:
Is there a reason that the KPIs and their rules weren't numbered? Otherwise I would add them.

As usual: feedback is greatly appreciated!

### CODE REVIEW

File: kpi/core/title.adoc [hunk 1]
```diff
@@ -6,67 +6,50 @@
...
-typically define which parameter is contained in the product, and from which
-instrument or instrument type it originates. For example, "AMSR-2 Sea Surface
-Temperature” or “SLSTR L1B radiances and brightness temperatures”. Please see
-the Guidance on WIS Discovery Metadatafootnote:[https://library.wmo.int/index.php?lvl=notice_display&id=6856#.X2TSLZMzYUF]
-for more information.
+The title should be as specific as possible. For example, if the product only contains one parameter, this can be stated in the title; however, if the product contains numerous parameters, then a more general term should be used in the title, and the parameters stated elsewhere in the metadata record (description, themes, keywords, etc.).
+
+For a satellite product offering one main data parameter, the title will typically define which parameter is contained in the product, and from which instrument or instrument type it originates. For example, "AMSR-2 Sea Surface Temperature” or “SLSTR L1B radiances and brightness temperatures”. Please see the Guidance on WIS Discovery Metadatafootnote:[https://library.wmo.int/index.php?lvl=notice_display&id=6856#.X2TSLZMzYUF] for more information.
```
Comment (amilan17) [inline kpi/core/title.adoc:54]: We should remove this reference to the Guidance on WIS Discovery Metadata in the library, because it's for WCMP1.3

**Review by amilan17 — APPROVED** [review]
I approve of all the changes introduced, but left a couple comments for some other minor revisions.

**Comment by jsieland** [comment]
I have updated the PR according to Anna's feedback so please review. Thanks @amilan17!

**Comment by tomkralidis** [comment]
TT-WISMD 2023-05-22: TT agrees to merge.
